study.md

```diff
@@ -40,14 +40,14 @@ What does an exclamation point after a method usually signify?  e.g.,
 `my_string.capitalize!`
 
 ```md
-<!-- your answer here -->
+It means it wants to capitalize the actual string (or element) instead of making a copy of the element with the action executed.
 ```
 
 ## Instantiation
 How do you create an instance of a class in Ruby?
 
 ```ruby
-# your answer here
+`class` + <name_of_class>
 ```
```

Comment:
I think I see what you going for, but the answer is : 
```ruby
my_instance = MyClass.new
```

Does that make sense after todays talks?